ecs-cli/modules/cli/compose/entity/service/service_helper.go

```diff
@@ -15,17 +15,74 @@ package service
 
 import (
 	"fmt"
+	"sort"
+	"time"
 
 	log "github.com/Sirupsen/logrus"
+	ecscli "github.com/aws/amazon-ecs-cli/ecs-cli/modules/commands"
 	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/utils/waiters"
 	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ecs"
 )
 
+// TimeOutUpdateService is the time that the CLI will wait to check if the
+// count of running tasks is changing. If it has not changed then an error is thrown
+// after TimeOutUpdateService minutes
+const TimeOutUpdateService = 5
+const PrintOnlyNewEvents = 10
+
+// serviceEvents is a wrapper for []*ecs.ServiceEvent
+// that allows us to reverse it
+type eventSorter []*ecs.ServiceEvent
+
+func (s eventSorter) Len() int {
+	return len(s)
+}
+func (s eventSorter) Swap(i, j int) {
```

Comment:
This is how you create a custom sorting order in golang. Its like building a custom comparator in Java.